src/main/resources/config/application.properties

```diff
@@ -48,6 +48,13 @@ idam.oauth2.client.secret=${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
 idam.oauth2.redirectUrl=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
 idam.oauth2.jwkUrl=${IDAM_API_JWK_URL:http://localhost:5000/jwks}
 
+auth.idam.client.baseUrl=${IDAM_API_URL:http://localhost:5000}
```

Comment:
Now you have added this config, do we still need the config above from line 40 -49? Abit annoying we have duplicate config otherwise and would be good to clean up.

Reply:
Yes we require 40-49 config, spring-security is expecting auth.idam.client.baseUrl formate